tests/test_code_gen.py

```diff
@@ -641,6 +641,300 @@ def test_compile_types(self):
             if code.strip() != dsttxt.strip():
                 self.assertEqual('(%s)' % code.strip(), dsttxt.strip())
 
+    @unittest.skipUnless(sys.version_info >= (3, 10, 0),
+                         "match statement introduced in Python 3.10")
+
```

Comment:
Nitpick: Can we remove the extra newline here and below?

Reply:
Fixed, thanks!

Reply:
No worries! I wasn't super clear anyway :)